GetIntoTeachingApi/Controllers/CandidatesController.cs

```diff
@@ -36,6 +46,13 @@ public IActionResult CreateAccessToken([FromBody, SwaggerRequestBody("Candidate
                 return BadRequest(this.ModelState);
             }
 
+            if (true) // TODO:

```

Comment:
is this // TODO still supposed to be here?

Reply:
Yeah:

> There is a // TODO: in the action which will later be the call out to the CRM to find a matching candidate (so we only send a PIN code if there is a matching candidate in the CRM).

Reply:
cool